basedecorator.rb

```diff
@@ -0,0 +1,11 @@
+require 'namable'
+class BaseDecorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly add `attr_accessor :nameable` and you can remove the `super`, it would be nice 🙏🏼 
```ruby
class Decorator < Nameable
    attr_accessor :nameable
  
    def initialize(nameable)
      @nameable = nameable
    end
  

    def correct_name
      @nameable.correct_name
    end
end
```

Reply:
@mukhammadrizooff 
how are you? Hope your fine
the reason why l used the super keyword is because its required when am configuring linters, without it the linters don't pass